ddd-koans/Value Object/Extra Task - Sealed Class & Functional Approach/task.html

```diff
@@ -0,0 +1,28 @@
+<html>
+<p>
+    You may have noticed that in Self Validation task we throw exception -> this behaviour breaks the flow. Please look at
```

Comment:
I would rather put a different wording here to encourage the user to run a snippet to see the broken flow. Sth like:

```
Please take a look at the code in the `main` function and run it. What has happened? Has anything been printed?

...
Sadly, the flow of this simple logic was broken because current implementation throws an exception. It may have disastrous effects if not handled properly.

This is not the best thing to do so we would change the way we create TimeSignatures to be sure that exceptions are not thrown.


(HERE: rest of text starting from `First of all, ...`
```

what do you think?